themes/default/layouts/webinars/single.html

```diff
@@ -75,7 +75,11 @@ <h4 class="text-orange">{{ $formHeader }}</h4>
                                 sessions="{{ .Params.multiple | jsonify }}"
                             ></pulumi-webinar-form-select>
                         {{ else }}
-                            <pulumi-hubspot-form form-id="{{ .Params.form.hubspot_form_id }}" go-to-webinar-key="{{ .Params.form.gotowebinar_key }}"></pulumi-hubspot-form>
+                            <pulumi-hubspot-form
+                                form-id="{{ .Params.form.hubspot_form_id }}"
+                                go-to-webinar-key="{{ .Params.form.gotowebinar_key }}"
```

Comment:
Is it gonna scream about this missing parameter?

Reply:
should be fine. that parameter was there before. it only looks new because prettier decided to reformat this thing.